### ISSUE TITLE
How can use predict_proba in the onnx？

### ISSUE THREAD

**Issue by yuanluw** [huggingface/setfit#359]
Hello, thanks for your work, it's very helpful.
But I encountered some issues during the usage. I want to deploy my model using ONNX, but the code provided by you directly outputs the classification labels. Is there any way for me to obtain the predicted probability values? like model.predict_proba()
The code provided by you is as follows:
`
 tokenizer = AutoTokenizer.from_pretrained("")
    inputs = tokenizer(
        input_text,
        padding=True,
        truncation=True,
        return_attention_mask=True,
        return_token_type_ids=True,
        return_tensors="np",
    )

session = onnxruntime.InferenceSession(output_path)

onnx_preds = session.run(None, dict(inputs))[0]
`

By the way, I want to obtain the predicted probability values mainly because besides the predefined classes, we also have an "other" class. However, the scope of the "other" class is very broad, and it is difficult for me to provide enough samples. Therefore, I want to judge it based on the probability value. Do you have any good suggestions for this issue?

**Comment by bogedy** [comment]
Just change the 0 at the end of `session.run(None, dict(inputs))[0]` to 1. It's already returning both :)

**Comment by tomaarsen** [comment]
I think that might solve it indeed! Let us know if that doesn't work :)
I'll close this in the meantime.

- Tom Aarsen

**Comment by yuanluw** [comment]
> Just change the 0 at the end of `session.run(None, dict(inputs))[0]` to 1. It's already returning both :)

thanks!
